yt_dlp/extractor/bilibili.py

```diff
@@ -316,20 +357,42 @@ def _real_extract(self, url):
 
         cid = traverse_obj(video_data, ('pages', part_id - 1, 'cid')) if part_id else video_data.get('cid')
 
+        festival_info = {}
+        if is_festival:
+            play_info = self._download_json(
+                'https://api.bilibili.com/x/player/playurl', video_id,
+                query={'bvid': video_id, 'cid': cid, 'fnval': 4048},
+                note='Extracting festival video formats')['data']
+
+            festival_info.update({
+                'duration': float_or_none(play_info.get('timelength'), scale=1000),
+                **traverse_obj(initial_state, {
+                    'uploader': ('videoInfo', 'upName'),
+                    'uploader_id': ('videoInfo', 'upMid', {str_or_none}),
+                    'like_count': ('videoStatus', 'like', {int_or_none}),
+                    'thumbnail': ('sectionEpisodes', lambda _, v: v['bvid'] == video_id, 'cover'),
+                }, get_all=False),
+            })
```

Comment:
just noticed duration was duplicated
```suggestion
            festival_info = traverse_obj(initial_state, {
                'uploader': ('videoInfo', 'upName'),
                'uploader_id': ('videoInfo', 'upMid', {str_or_none}),
                'like_count': ('videoStatus', 'like', {int_or_none}),
                'thumbnail': ('sectionEpisodes', lambda _, v: v['bvid'] == video_id, 'cover'),
            }, get_all=False)
```